Makefile

```diff
@@ -294,20 +301,30 @@ docs_serve:
 # Test #
 ########
 
-subsystem-run: test subsystem-clean
+subsystem-run: | test-rest-api subsystem-clean enable-kube-api-for-subsystem test-kube-api subsystem-clean
+
+test-rest-api:
+	$(MAKE) _run_test AUTH_TYPE=rhsso
 
-test:
```

Comment:
Please keep `test` as it is used by [Prow](https://github.com/openshift/assisted-service/blob/f30d42d8d76f66bec73f9e830b8f0b27996eb2b5/Makefile#L349-L351)
Otherwise, it would break CI.